README.md

```
@@ -8,21 +8,162 @@
+   dotnet ef database update
+   ```
+
+5. **Start the Backend Server**:
+   ```bash
+   dotnet run
+   ```
+
+### Set Up the Frontend
+
+1. **Navigate to the Frontend Directory**:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)